### PR TITLE
Bumping version for CRAN release

### PR DESCRIPTION
**Pull request by heemanshusuri**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# azuremlsdk 0.8.0
+## New features
+- Foreach backend for distirbuted training and batch inferencing.
```
Comment (diondrapeck) [inline NEWS.md:3]: typo: *distributed training

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+- R Section in Environment Definition.
+- Added Azure Data Lake Gen2 Datastore support.
+
+## Bug fixes
+- Fix issues with Dataset creation and usage.
+- Fix Interactive Authentication
+
```
Comment (diondrapeck) [inline NEWS.md:10]: we should also mention that we've removed the widget function in this version

File: DESCRIPTION [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Package: azuremlsdk
 Type: Package
 Title: Interface to the 'Azure Machine Learning' 'SDK'
-Version: 0.6.85
+Version: 0.8.0
```
Comment (diondrapeck) [inline DESCRIPTION:4]: why did we jump from 0.6.85 to 0.8.0 rather than 0.7.0? I may have missed a conversation about this.

Comment (heemanshusuri) [reply DESCRIPTION:4]: we are using python sdk==1.8.0 as default version in install_azureml(), so, I made it 0.8.0 to be consistent with it.

File: NEWS.md (hunk 2 above)
Comment (heemanshusuri) [reply NEWS.md:10]: added

File: NEWS.md (hunk 1 above)
Comment (heemanshusuri) [reply NEWS.md]: thanks for catching it

**Pull request merged**